src/modules/emotion.command.js

```diff
@@ -0,0 +1,23 @@
+const {text, pictures} = require('./emotion-list.json');
+
+// TODO: любая эмоция может быть командой. любая команда может быть эмоцией
+// TODO: получаю эмоцию-команду и беру из пула сообщение и картинку
+const emotionCommand = async function(response, { username, input, i18n }) {
+    if (input[0] !== '!') {
+        return response;
+    }
+
+    const command = input.split(` `)[0];
+
+    const emotion = i18n(command);
+    const message = text[emotion];
```

Comment:
extract it by keyword from i18n like
```
const message = i18n(`emotion_${key}`, { ...someUserData });
```

and in i18n.csv
```
emotion_sad, "@${userId} is sad"
emotion_sad, "@${userId} saaadddd"
emotion_sad, "@${userId} sad!"
emotion_sad, "sad... saddd.... SADDD!!! AAA @${userId}"
```

u have randomizer from i18n and interpolation with context data! its simple just use it <3

Reply:
How I can get `userId`?

Reply:
u has
```
const { id } = context;
```

in module scope))